### ISSUE TITLE
Carthage build fails on Apple M1

### ISSUE THREAD

**Issue by dmitrym0** [dropbox/SwiftyDropbox#307]
Hi folks,

- Big Sur, 11.2
- Mabook Air M1

- Repo to reproduce this is [here](https://github.com/dmitrym0/dropbox-test), I'm using an empty SwiftUI project.

When using Carthage instructions I get the following:

```
~/w/d/dropbox-test ❯❯❯ carthage update --use-xcframeworks
*** Fetching SwiftyDropbox
*** Fetching Alamofire
*** Checking out Alamofire at "4.9.1"
*** Checking out SwiftyDropbox at "6.0.3"
*** xcodebuild output can be found in /var/folders/zv/rmxrv3kd3qz4prhv0dp63klc0000gn/T/carthage-xcodebuild.NfgdpQ.log
*** Building scheme "Alamofire macOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire watchOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire tvOS" in Alamofire.xcworkspace
*** Building scheme "SwiftyDropbox iOS" in SwiftyDropbox.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/dmitry/workspace/dropbox-test/dropbox-test/Carthage/Checkouts/SwiftyDropbox/Source/SwiftyDropbox/SwiftyDropbox.xcodeproj -scheme SwiftyDropbox\ iOS -configuration Release -derivedDataPath /Users/dmitry/Library/Caches/org.carthage.CarthageKit/DerivedData/12.4_12D4e/SwiftyDropbox/6.0.3 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive VALIDATE_WORKSPACE=NO -archivePath /var/folders/zv/rmxrv3kd3qz4prhv0dp63klc0000gn/T/SwiftyDropbox SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/dmitry/workspace/dropbox-test/dropbox-test/Carthage/Checkouts/SwiftyDropbox)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/zv/rmxrv3kd3qz4prhv0dp63klc0000gn/T/carthage-xcodebuild.NfgdpQ.log
```

**Comment by greg-db** [comment]
Thanks for the report. We'll look into it.

For reference, can you also share:
- the version number of Carthage you have installed
- the version number of Xcode you're using
- the carthage-xcodebuild.NfgdpQ.log file

**Comment by dmitrym0** [comment]
Hi Greg:

```
~ ❯❯❯ brew info carthage
carthage: stable 0.37.0 (bottled), HEAD
```

- Xcode 12.4 12D4e
- [carthage-xcodebuild.NfgdpQ.log](https://github.com/dropbox/SwiftyDropbox/files/6123594/carthage-xcodebuild.NfgdpQ.log)

Thank you!

**Comment by greg-db** [comment]
Thanks!

**Comment by greg-db** [comment]
This should be fixed with [v7.0.0](https://github.com/dropbox/SwiftyDropbox/releases/tag/7.0.0). Please give that a try and let us know if you're still seeing any issues. Thanks!

**Comment by dmitrym0** [comment]
Thanks, can confirm that it builds!
